app/src/main/res/layout/fragment_market_selected.xml

```diff
@@ -0,0 +1,69 @@
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:theme="@style/ThemeOverlay.Hedvig.MarketingActivity"
```

Comment:
Should this `ThemeOverlay` be applied to the entire `Activity`?

Reply:
It is applied to the activity but it was still needed to be applied to this fragment